core/src/organization/member-restriction/member-restriction.resolver.ts

```diff
@@ -1,24 +1,29 @@
-import {UseGuards} from "@nestjs/common";
+// import {UseGuards} from "@nestjs/common";
+// import {MLE_OrganizationTeam} from "../../database/mledb";
+// import {MLEOrganizationTeamGuard} from "../../mledb";
+import {Inject} from "@nestjs/common";
 import {
-    Args, Int, Mutation, Query, ResolveField, Resolver, Root,
+    Args, Int, Mutation, Query, ResolveField, Resolver, Root, Subscription,
 } from "@nestjs/graphql";
+import {PubSub} from "apollo-server-express";
 
 import type {Member} from "../../database";
 import {MemberRestriction, MemberRestrictionType} from "../../database";
-import {MLE_OrganizationTeam} from "../../database/mledb";
-import {MLEOrganizationTeamGuard} from "../../mledb";
+import {MemberPubSub} from "../constants";
 import {MemberService} from "../member/member.service";
+import {MemberRestrictionEvent} from "./member.types";
 import {MemberRestrictionService} from "./member-restriction.service";
 
 @Resolver(() => MemberRestriction)
 export class MemberRestrictionResolver {
     constructor(
         private readonly memberRestrictionService: MemberRestrictionService,
         private readonly memberService: MemberService,
+        @Inject(MemberPubSub) private readonly pubSub: PubSub,
     ) {}
 
     @Query(() => [MemberRestriction])
-    @UseGuards(MLEOrganizationTeamGuard(MLE_OrganizationTeam.MLEDB_ADMIN))
+    // @UseGuards(MLEOrganizationTeamGuard(MLE_OrganizationTeam.MLEDB_ADMIN))
```

Comment:
Is this needed here; is this an admin only thing?

Reply:
So, we went back and forth on this in VC. It's only used by the admin interface currently, so it's already gated behind admin team membership. That said, I don't think that putting a guard on it in the backend will hurt anything, we don't expect to have a ton of restrictions being made so no worries about performance. I commented it out because of the bug in orgTeams payloads.

Reply:
Problem being, anyone can pull up some request making application and make a gql request to the endpoint without using the web client.

Reply:
https://app.clickup.com/t/2ecmadf